### PR TITLE
v0.16.0

### PR DESCRIPTION
**Pull request by islandbitcoin**
changed ssl to `true` for one of the ports in the interface, and merged the js-properties branch into this.
bumped the version to v16.

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 id: lnd
 title: LND
-version: 0.15.5.2
+version: 0.16.0
 release-notes: |-
-  * LND v0.15.5 [Release Notes](https://github.com/lightningnetwork/lnd/blob/v0-15-5-branch/docs/release-notes/release-notes-0.15.5.md)
-  * Add new action for migration from Umbrel v0.5 and update action for migration from Umbrel v0.4
-  * Updated service name from Lightning Network Daemon to LND
+  * LND v0.16.0 [Release Notes](https://github.com/lightningnetwork/lnd/blob/master/docs/release-notes/release-notes-0.16.0.md)
+  * Start9 packaging performance updates and bigfixes
```
Comment (elvece) [inline manifest.yaml:6]:
```suggestion
  * Start9 packaging performance updates and bugfixes
```

File: scripts/services/properties.ts [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    }),
+    effects.readFile({
+      volumeId: "main",
+      path: "start9/control.cert.pem.base64url",
+    }),
+  ]);
+
+  try {
+    const nodeInfo = await effects.fetch(
+      "https://lnd.embassy:8080/v1/getinfo",
```
Comment (elvece) [inline scripts/services/properties.ts:77]:
```suggestion
      "https://lnd.embassy:8081/v1/getinfo",
```

File: scripts/services/properties.ts [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+          value: `${nodeInfoJson.identity_pubkey}@${peerTorAddress}:9735`,
+          description:
+            "Give this to others to allow them to add your LND node as a peer",
+          copyable: true,
+          qr: true,
+          masked: false,
+        },
+        "LND Connect gRPC URL": {
+          type: "string",
+          value: `lndconnect://${controlTorAddress}:10009?cert=${cert}&macaroon=${macaroonBase64URL}`,
```
Comment (elvece) [inline scripts/services/properties.ts:141]: does this port need to change too?

File: scripts/services/properties.ts [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+          description:
+            "Use this for other applications that require a gRPC connection",
+          copyable: true,
+          qr: true,
+          masked: true,
+        },
+        "LND Connect REST URL": {
+          type: "string",
+          value: `lndconnect://${controlTorAddress}:8080?cert=${cert}&macaroon=${macaroonBase64URL}`,
```
Comment (elvece) [inline scripts/services/properties.ts:150]: same here?

File: configurator/src/main.rs [hunk 5]
```diff
@@ -809,23 +765,24 @@ fn get_stats(
...
             .arg("--header")
             .arg(format!("Grpc-Metadata-macaroon: {}", mac_encoded))
-            .arg("https://lnd.embassy:8080/v1/getinfo")
+            .arg("http://localhost:8081/v1/getinfo")
             .output()?
             .stdout,
     )
     .or::<anyhow::Error>(Ok(None))?;
+
+    println!("writing stats.yaml...");
```
Comment (elvece) [inline configurator/src/main.rs:785]: why do we need to write this file if properties are now using JS procedures?

File: scripts/services/properties.ts (hunk 3 above)
Comment (islandbitcoin) [reply scripts/services/properties.ts:141]: not sure. this port worked when I tested it as-is

File: configurator/src/main.rs (hunk 5 above)
Comment (islandbitcoin) [reply configurator/src/main.rs:830]: is stats.yaml used anywhere else in the OS or for the wrapper? if not, I can comment it out

File: manifest.yaml (hunk 1 above)
Comment (elvece) [reply manifest.yaml:6]: should this really be "bigfixes"?

File: configurator/src/main.rs [hunk 6]
```diff
@@ -698,8 +683,15 @@ fn main() -> Result<(), anyhow::Error> {
...
     }
- 
-    if true { 
+
+    // write properties once
+    // properties(
+    //     control_tor_address.clone(),
+    //     peer_tor_address.clone(),
+    //     alias.clone(),
+    // )?;
```
Comment (elvece) [inline configurator/src/main.rs:692]: remove me

File: configurator/src/main.rs [hunk 7]
```diff
@@ -801,201 +798,226 @@ fn main() -> Result<(), anyhow::Error> {
...
-                        },
-                    },
-                };
-                stats
-            }
-        }
-    };
-    Ok(stats)
-}
+// fn get_stats(
```
Comment (elvece) [inline configurator/src/main.rs:806]: remove all commented out code unless its needed. it can always be referenced in the git history

**Review by elvece — CHANGES_REQUESTED** [review]
Few cleanup comments

**Review by elvece — APPROVED** [review]

**Pull request merged**